### DISCUSSION TITLE
Signalk Cloud: Server certificate expired

### DISCUSSION THREAD

**Discussion by Sveinos** [sbender9/signalk-cloud#26] [FAQ]
Hi,

I have been using Signalk Cloud for some time. Some days ago the plugin could not connect to the server, and reports the error: signalk-cloud:Error connecting to cloud server certificate has expired.

Any suggestions ? :-)

**Comment by SL3Github** [comment]
HI, i am getting the same error and cannot use WilhelmSK

**Reply by Sveinos** [reply]
Hi, As far as I can see, the problem is the connection between the SignalK plugin and the cloud server. If I make a connection from WilhelmSK to Victron VRM (Cloud), the Victron data is ok in WilhelmSK.

I have tried to find some kind of support, but no luck so far...
